### ISSUE TITLE
Optional internet access

### ISSUE THREAD

**Issue by Naheel-Azawy** [ErrorNoInternet/ok#2]
ok, so I'm a bit paranoid of network traffic.
This is a feature request, is it possible to separate network related actions?
Something like keeping such tasks in a separate file and somehow optionally import it at compile time.
ok?

**Comment by ErrorNoInternet** [comment]
HTTP requests are only sent if you use a specific sub-command (`ok list`, `ok join`, `ok update`, etc)

**Comment by Naheel-Azawy** [comment]
Yeah but I thought why have that in the binary if I don't need them.
I have already commented those parts before compiling which gets the job done for me.
If this sounds like too much unnecessary work feel free to close this issue and it will be ok

**Comment by ErrorNoInternet** [comment]
I'll add this anyway, to clean up some terrible code and learn more about Go build tags

**Comment by Naheel-Azawy** [comment]
ok

**Comment by ErrorNoInternet** [comment]
Should I rewrite the whole thing in Rust? :eyes:

**Comment by Naheel-Azawy** [comment]
Hmm, I was thinking of a C rewrite but unfortunately I have to get back to work.
Rust would be nice as well. But even Go is okish for such an ok application.
